docs/dotnet-runtime.md

```diff
@@ -0,0 +1,34 @@
+---
+id: dotnet-runtime
+title: Scope .NET Agent runtime instrumentation
+sidebar_label: Runtime instrumentation
+---
+
+## Runtime instrumentation
+
+In order to see trace information from your .NET service on integration and end-to-end tests,
+you need to use the Scope agent to instrument your running service.
+
+This service might run, for example, in a container on CI, or in a QA/staging environment.
+
+To instrument in runtime, prefix your service startup command with `scope-run`. For example:
+                          
+```bash
+scope-run dotnet run
+```
+
+This will autoinstrument your application at runtime and will produce tracing information if it receives
+a request from an instrumented test or application.
+
+## Adding code path information for runtime binaries
+
+Enabling the code path information of runtime applications requires to patch the assemblies 
```

Comment:
Does this also apply to running tests with coverage information?

Reply:
No, when running tests this process is done by a test collector. The test collector is automatically added by the scope-run command.